distribution/docs/src/main/resources/_contents/_core-concepts/metadata-intro-contents.adoc

```diff
@@ -0,0 +1,56 @@
+
+== Understanding Metadata and Metacards
+
+Metadata is information about a resource, organized into a schema to make it possible to search against.
+The ${ddf-catalog} stores this metadata and allows access to it.
+Metacards are single instances of metadata, representing a single record, in the Metadata Catalog (MDC).
+Metacards follow one of several schemas to ensure reliable, accurate, and complete metadata.
+Essentially, Metacards function as containers of metadata.
+
+=== Populating Metacards During Ingest
+
+Upon ingest, a metacard transformer will read the data from the ingested file and populate the fields of the metacard.
+Exactly how this is accomplished depends on the origin of the data, but most fields (except id) are imported directly.
+
+=== Searching Metadata
+
+${branding} provides the capability to search the Metadata Catalog (MDC) for metadata.
+There are a number of different types of searches that can be performed on the MDC, and these searches are accessed using one of several interfaces.
+This section provides a very high level overview of introductory concepts of searching with ${branding}.
+These concepts are expanded upon in later sections.
+
+==== Search Types
+
+There are four basic types of metadata search.
+Additionally, any of the types can be combined to create a compound search.
+
+===== Contextual Search
+
+A contextual search is used when searching for textual information.
+It is similar to a Google search over the metadata contained in the MDC.
+Contextual searches may use wildcards, logical operators, and approximate matches.
+
+===== Spatial Search
+
+A spatial search is used for Area of Interest (AOI) searches.
+Polygon and point radius searches are supported.
+Specifically, the spatial search looks at the metacards' location attribute and coordinates are specified in *WGS 84* decimal degrees.
+
+===== Temporal Search
+
+A temporal search finds information from a specific time range.
+Two types of temporal searches are supported: _relative_ and _absolute_.
+Relative searches contain an offset from the current time, while absolute searches contain a start and an end timestamp.
+Temporal searches can look at the effective date attribute or the modified date.
```

Comment:
`Effective` is a weird concept that we're trying to get rid of. It was never very well-defined and has been deprecated as part of the expanded taxonomy effort. Favor `created` and `modified` explicitly instead. See https://codice.atlassian.net/wiki/display/DDF/%5Bdeprecated%5D+effective (we should probably update the javadoc as well)